coldfront/plugins/qumulo/templates/email/invalid_users.txt

```diff
@@ -0,0 +1,9 @@
+Notice for the following allocation:
+{{allocation_url}}
+
+The following users were not found and could not be granted {{ access_type }} access:
+{% for user in invalid_users %}
+{{ user }}
+{% endfor %}
+
+Please inspect and correct as needed.  The allocation will still be created or updated as normal, including all valid users.
```

Comment:
I would consider listing just the groups, rather than the users. In most cases that would mean only showing 2 groups; the RO and RW. But once we start nesting groups it might make things cleaner than listing all WUSTL keys effected.

Reply:
Not sure I understand, @gunnarwustl .  Also this email is currently one for RO, one for RW, not both together.  It was just easier to work in that way.

@peaches7083 if we add good user's here, we can basically just send it out everytime as a results email.

I guess it just depends on if we want to always send out this email, or just notify of issues.

Thoughts?